INSTALL.md

```diff
@@ -156,17 +156,14 @@ $ sudo make clean-shm
 
 ## Porting to different platforms
 
-If you need to port Gophernicus to a new platform, please take
-a look at gophernicus.h which has a bunch of `HAVE_*` `#defines`.
-Fiddling with those usually makes it possible to compile a working
-server. If you succeed in compiling Gophernicus to a new
-platform please send the patches to 
-<gophernicus at gophernicus dot org> so we can include them into
-the next release.
+If you need to port Gophernicus to a new platform, please take a look at
+gophernicus.h which has a bunch of `#define HAVE_...` Fiddling with those
+usually makes it possible to compile a working server.
+If you succeed in compiling Gophernicus to a new platform please send
+the patches to <gophernicus at gophernicus dot org> so we can include
+them into the next release.
```

Comment:
Maybe say make a PR instead?

Reply:
Yep, good idea! I added that, as I did not want to put it *instead of sending patch* by e-mail, as we should not assume everybody joining development will have a Github account (upfront).

Reply:
Yep, looks great!